time sent_time
geometry_msgs/Pose2D tracking_pose
float64 tracking_speed
bool reset_sim
bool path_planner_terminate
bool end_of_action
float64 action_progress
